docs/interfaces/webui-if.rst

```diff
@@ -71,6 +76,49 @@ visit :ref:`users`.
 With the Determined Enterprise Edition, you can also create and manage user groups. To learn more,
 visit :ref:`rbac`.
 
+************************
+ Managing User Settings
+************************
+
+User settings allow you to manage profile settings, preferences, and shortcuts. You can also toggle
+experimental features on or off and access advanced features.
+
+To view user settings:
+
+-  Select your profile name in the upper left corner and choose **Settings**.
+
+To change user settings:
+
+-  Select the edit icon.
+-  Make changes to the setting.
+-  Confirm the changes by selecting the checkmark.
+
+To revert to default settings:
+
+-  Navigate to the Advanced section of the user settings.
+-  Select **I know what I'm doing**.
+-  Select **Reset to Default**.
+-  Confirm you want to reset all user settings to their default values.
+
+****************************************
+ Selecting a Table Density (Row Height)
+****************************************
+
+In the Preferences section of your user settings, you can set the table density so that the rows are
+shorter or taller.
+
+******************************************
+ Toggling Experimental Features On or Off
+******************************************
+
+In the Experimental section of your user settings, you can turn experimental features on or off.
+However, if you don't know what the feature is referring to or the possible impact, you likely
+should not turn it on.
+
+.. caution::
+
+   Experimental features can be changed or removed at any time.
+
```

Comment:
@ClaireNeveu i don't see the feedback anymore, but the edit has been applied.

Reply:
It is important to say that these are pre-release features.

Reply:
@ClaireNeveu this should now be resolved